politeiawww/cmd/shared/client.go

```diff
@@ -1581,29 +1581,29 @@ func (c *Client) EditUser(eu *www.EditUser) (*www.EditUserReply, error) {
 	return &eur, nil
 }
 
-// AuthorizeVote authorizes the voting period for the specified proposal using
+// VoteAuthorize authorizes the voting period for the specified proposal using
 // the logged in user.
-func (c *Client) AuthorizeVote(av *www.AuthorizeVote) (*www.AuthorizeVoteReply, error) {
-	responseBody, err := c.makeRequest(http.MethodPost, www.PoliteiaWWWAPIRoute,
-		www.RouteAuthorizeVote, av)
+func (c *Client) VoteAuthorize(va *pi.VoteAuthorize) (*pi.VoteAuthorizeReply, error) {
```

Comment:
Can we update these client alls to accept structs instead of pointers?  There's not really a reason to pass a struct to function as a pointer unless you're planning on editing it.

Reply:
surething